src/Cron_Command.php

```diff
@@ -85,27 +85,110 @@ public function create( $args, $assoc_args ) {
 		$schedule = EE\Utils\get_flag_value( $assoc_args, 'schedule' );
 
 		if ( '@' !== substr( trim( $schedule ), 0, 1 ) ) {
-			$schedule_length = strlen( explode( ' ', trim( $schedule ) ) );
-			if ( $schedule_length <= 5 ) {
-				$schedule = '0 ' . trim( $schedule );
+			$schedule_length = strlen( implode( explode( ' ', trim( $schedule ) ) ) );
+			if ( 5 !== $schedule_length ) {
```

Comment:
Can not use this check. It will fail for schedules having `/` or `double digits` in them, like `30 08 10 06 *`, whose string length will increase. 

It is better to count the number of elements, remove imploding for that, and to handle accidental multiple spaces in between which will cause empty elements in array after exploding you can use the count after applying `array_filter` with `trim` on the array.